sentry_sdk/integrations/gcp.py

```diff
@@ -159,22 +159,22 @@ def _get_google_cloud_logs_url(initial_time):
     Returns:
         str -- Google Cloud Logs Console URL to logs.
     """
-    hour_ago = initial_time - timedelta(hours=1)
+    hour_ago = final_time - timedelta(hours=1)
 
     url = (
         "https://console.cloud.google.com/logs/viewer?project={project}&resource=cloud_function"
         "%2Ffunction_name%2F{function_name}%2Fregion%2F{region}&minLogLevel=0&expandAll=false"
-        "&timestamp={initial_time}&customFacets=&limitCustomFacetWidth=true"
-        "&dateRangeStart={timestamp_start}&dateRangeEnd={timestamp_end}"
-        "&interval=PT1H&scrollTimestamp={timestamp_current}"
+        "&timestamp={initial_time}Z&customFacets=&limitCustomFacetWidth=true"
+        "&dateRangeStart={timestamp_start}Z&dateRangeEnd={timestamp_end}Z"
+        "&interval=PT1H&scrollTimestamp={timestamp_current}Z"
     ).format(
         project=environ.get("GCP_PROJECT"),
         function_name=environ.get("FUNCTION_NAME"),
         region=environ.get("FUNCTION_REGION"),
-        initial_time=initial_time,
+        initial_time=final_time,
```

Comment:
Why is `initial_time` set to `final_time`?!

Reply:
I'll make this change, was left out in the last changes that were made.